docs/requirements.txt

```diff
@@ -4,4 +4,5 @@
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
--e .[all]
+-e .[docs]
+-e .[kubernetes]
```

Comment:
Question: Why is `kubernetes` necessary to generate docs?

Reply:
There was a warning when I didn't include `kubernetes`

```
WARNING: autodoc: failed to import module 'api_client' from 
module 'reana_commons.k8s'; the following exception was raised:
No module named 'kubernetes'
```

Reply:
When you rebased against `maint-0.7`, please delete all the extra commits this brought, and keep only yours.